pkg/machines/app.jsx

```diff
@@ -154,6 +154,7 @@ class App extends React.Component {
                     config={config}
                     ui={ui}
                     libvirtVersion={systemInfo.libvirtVersion}
+                    snapshotsAPIsAvailability={systemInfo.snapshotsAPIsAvailability}
```

Comment:
Hard to read - "Is Availability"? how about "haveSnapshots"?

Reply:
I mean that's not really about VM having any snapshots. It's about user having new enough version of libvirt-dbus with snapshots APIs. But sure I can change the name

Reply:
Right, "haveSnapshotAPI" makes more sense then, or "snapshotAPIAvailable"